plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/WooPayments.php

```diff
@@ -133,11 +133,11 @@ protected function get_incentives( string $country_code ): array {
 			// Store locale, e.g. `en_US`.
 			'locale'       => get_locale(),
 			// WooCommerce store active for duration in seconds.
-			'active_for'   => WCAdminHelper::get_wcadmin_active_for_in_seconds(),
-			'has_orders'   => $this->has_orders(),
+			'active_for'   => 8000000,
```

Comment:
Oops, I think values from the patch in the PR description got checked in.

Reply:
Great catch @anu-rock - removed it :)